Bio/Align/__init__.py

```diff
@@ -19,7 +19,14 @@
 from Bio.SeqRecord import SeqRecord, _RestrictedDict
 from Bio import Alphabet
 
-from Bio.Align import _aligners
+try:
+    from Bio.Align import _aligners
+except ImportError as e:
+    new_exc = ImportError("""{}: you should not import directly from the
```

Comment:
Don't use a triple quote string, which inserts the new lines breaks as is. Use single quotes (which will auto-concatenate)